doc/source/data/transforming-datasets.rst

```diff
@@ -4,30 +4,12 @@
 Transforming Datasets
```

Comment:
How about keep map/filter/flat_map here? This is a more detailed page for APIs that can be used to transform datasets. Since we still have those in Dataset, we need to keep them in documentation. Maybe just add a "tip" or alert on performance.

Reply:
I added this as a tip (that we have map/filter/flat_map), but avoided keeping it in the main body.